Source/Controllers/Other/SPSQLiteHistoryManager.h

```diff
@@ -0,0 +1,34 @@
+//
+//  SPSQLiteHistoryManager.h
+//  Sequel Ace
+//
+//  Created by James on 17/11/2020.
+//  Copyright © 2020 Sequel-Ace. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <fmdb/FMDB.h>
```

Comment:
Do we need this in `.h` file or can we add it to the `m` file only and add `@class FMDatabaseQueue;` here in header file?

Reply:
probably not...